src/relayclient/GsnTestEnvironment.ts

```diff
@@ -66,8 +66,10 @@ class GsnTestEnvironmentClass {
       throw new Error('Failed to run a local Relay Server')
     }
 
-    const registerOptions = {
+    const registerOptions: RegisterOptions = {
       from,
+      sleepMs: 100,
```

Comment:
sleep/sleepcount is bad, and it doesn't support anythning by ganache.
better have "registrationTimeout (1 on ganache, 60 on real network)
and "retryTimeout" (0.1 on ganache, 10 sec on real network.)

Reply:
This PR was approved merged to `master` and is a backport (literally `git cherry-pick`) to the `release` branch.
It is too late to introduce changes to this commit. Please open a Jira issue to introduce changes to stake ownership mechanism.

Reply:
You mean: "I ignored your comments on master, and now backported it as I can ignore them again"